README.md

```diff
@@ -3,18 +3,21 @@ FreeRTOS offers feature stability with long term support (LTS) releases. FreeRTO
-| FreeRTOS-Plus-TCP     | 2.3.2   | 12/31/2022 | https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/V2.3.2                 |
-| coreMQTT              | 1.1.0   | 12/31/2022 | https://github.com/FreeRTOS/coreMQTT/tree/v1.1.0                          |
-| coreHTTP              | 2.0.0   | 12/31/2022 | https://github.com/FreeRTOS/coreHTTP/tree/v2.0.0                          |
-| corePKCS11            | 3.0.0   | 12/31/2022 | https://github.com/FreeRTOS/corePKCS11/tree/v3.0.0                        |
-| coreJSON              | 3.0.0   | 12/31/2022 | https://github.com/FreeRTOS/coreJSON/tree/v3.0.0                          |
-| backoffAlgorithm      | 1.0.0   | 12/31/2022 | https://github.com/FreeRTOS/backoffAlgorithm/tree/v1.0.0                  |
-| AWS IoT Device Shadow | 1.0.2   | 12/31/2022 | https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/tree/v1.0.2 |
+Libraries in this GitHub branch (also listed below) are part of the [FreeRTOS 202012.01 LTS](https://github.com/FreeRTOS/FreeRTOS-LTS/tree/202012-LTS) release. Learn more at https://freertos.org/lts-libraries.html.
+
+| Library                     | Version | LTS Until  | LTS Repo URL                                                                |
+|-------------------------    |---------|------------|---------------------------------------------------------------------------  |
+| FreeRTOS Kernel             | 10.4.3  | 03/31/2023 | https://github.com/FreeRTOS/FreeRTOS-Kernel/tree/V10.4.3                    |
+| FreeRTOS-Plus-TCP           | 2.3.2   | 03/31/2023 | https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/V2.3.2                   |
+| coreMQTT                    | 1.1.0   | 03/31/2023 | https://github.com/FreeRTOS/coreMQTT/tree/v1.1.0                            |
+| coreHTTP                    | 2.0.0   | 03/31/2023 | https://github.com/FreeRTOS/coreHTTP/tree/v2.0.0                            |
+| corePKCS11                  | 3.0.0   | 03/31/2023 | https://github.com/FreeRTOS/corePKCS11/tree/v3.0.0                          |
+| coreJSON                    | 3.0.0   | 03/31/2023 | https://github.com/FreeRTOS/coreJSON/tree/v3.0.0                            |
+| backoffAlgorithm            | 1.0.0   | 03/31/2023 | https://github.com/FreeRTOS/backoffAlgorithm/tree/v1.0.0                    |
+| AWS IoT Device Shadow       | 1.0.2   | 03/31/2023 | https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk/tree/v1.0.2   |
+| AWS IoT Device Defender     | 1.1.0   | 03/31/2023 | https://github.com/aws/Device-Defender-for-AWS-IoT-embedded-sdk/tree/v1.1.0 |
+| AWS IoT Jobs                | 1.1.0   | 03/31/2023 | https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/tree/v1.1.0            |
+| AWS IoT Over-the-air Update | 3.0.0   | 03/31/2023 | https://github.com/aws/ota-for-aws-iot-embedded-sdk/tree/v3.0.0             |
 
 ## FreeRTOS LTS Versioning and Patches
 
```

Comment:
We need to add a line here for

202012.01: Added AWS IoT Device Defender, IoT Jobs, IoT OTA to FreeRTOS 202012-LTS.

Reply:
Per separate conversation, I have added a CHANGELOG.md file for this information